content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -211,17 +211,14 @@ these credentials with others**. Each team member should go through this process
 individually.
 
 If you use multiple GDrive remotes, by default they will be sharing the same
-`.dvc/tmp/gdrive-user-credentials.json` credentials file. It can be overridden
-with the `gdrive_user_credentials_file` setting:
+`.dvc/tmp/gdrive-user-credentials.json` file. It can be overridden with the
+`gdrive_user_credentials_file` setting:
 
 ```dvc
 $ dvc remote modify myremote gdrive_user_credentials_file \
                     .dvc/tmp/myremote-credentials.json
 ```
 
-> If you edit the config file manually, value must be either an absolute path or
```

Comment:
why do we remove this?

Reply:
Big conceptual jump from `dvc remote modify` to "config file" which isn't explained in this doc. I.e. I don't think users will understand the note at all. We'd need to expand on that relationship and I don't think it's worth it. But lmk please